.jshintrc

```diff
@@ -115,6 +115,9 @@
         // Underscore.js globals
         "_",
 
+        // Backbone.js globals
+        "Backbone",
```

Comment:
This may be out of scope, but I think you're supposed to use requirejs, e.g.
```
require(["jquery", "backbone"], function($, Backbone) {
```
rather than assume the global is available. I know the existing code was not using requirejs and was already assuming a jquery global but you may as well clean it up rather than adding jshint exceptions.